scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -111,7 +111,7 @@ workflow GvsCreateFilterSet {
         intervals                  = SplitIntervals.interval_files[i],
         fq_alt_allele_table        = fq_alt_allele_table,
         alt_allele_table_timestamp = AltAlleleTableDatetimeCheck.last_modified_timestamp,
-        excess_alleles_threshold   = 1000000,
+        excess_alleles_threshold   = 100,
```

Comment:
Set Excess Alleles Threshold to 100!

Reply:
Do we need to document anything differently from this?

Reply:
Note for Beta (add in documentation)